rules/Php55/Rector/String_/StringClassNameToClassConstantRector.php

```diff
@@ -37,6 +38,13 @@ final class StringClassNameToClassConstantRector extends AbstractRector implemen
      */
     private array $classesToSkip = [];
 
+    private bool $isKeepFirstBackslashString = true;
+
```

Comment:
This should be false by default.

It seems a bit verbose, what about "shouldKeepPreslash"?

Reply:
sure, updated 👍 https://github.com/rectorphp/rector-src/pull/5354/commits/ed70ee9467ffbda5c0c35e01bf5728578bd94c01